articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -375,8 +375,8 @@ In addition to supporting any model from HuggingFace Transfomers and MMDetection
 
 Task |  model architectures | String literal syntax
 ---|----------|----------
-Image classification<br> (multi-class and multi-label)| **BEiT**: A Vision Transformer (ViT), which is a transformer encoder model (BERT-like). |[`microsoft/beit-base-patch16-224-pt22k-ft22k`](https://ml.azure.com/registries/azureml-preview/models/microsoft-beit-base-patch16-224-pt22k-ft22k/version/1)<br> [`google/vit-base-patch16-224`](https://ml.azure.com/registries/azureml-preview/models/google-vit-base-patch16-224/version/1)<br> [`facebook/deit-base-patch16-224`](https://ml.azure.com/registries/azureml-preview/models/facebook-deit-base-patch16-224/version/1)<br> [`microsoft/swinv2-base-patch4-window12-192-22k`](https://ml.azure.com/registries/azureml-preview/models/microsoft-swinv2-base-patch4-window12-192-22k/version/1)
-Object Detection | TODO |  TODO
+Image classification<br> (multi-class and multi-label)| **BEiT**: A Vision Transformer (ViT), which is a transformer encoder model (BERT-like). <br> **ViT**: A BERT-like transformer encoder model which is pretrained on a large collection of images in a supervised fashion. <br> **DeiT**: A more efficiently trained Vision Transformer (ViT) <br> **SwinV2**: A type of Vision Transformer used in both image classification and dense recognition tasks | `microsoft/beit-base-patch16-224-pt22k-ft22k`<br> `google/vit-base-patch16-224`<br> `facebook/deit-base-patch16-224`<br> `microsoft/swinv2-base-patch4-window12-192-22k`
```

Comment:
I am wondering if instead of providing the descriptions that we do here, which I honestly don't find very useful, we can just make the names of the models links to HF model cards, like https://huggingface.co/microsoft/swinv2-base-patch4-window12-192-22k

Reply:
I think that is a good idea! I don't think the descriptions are very useful either, there isn't much detail we can put in a table.

Reply:
Should we instead point them to azureml registry and/or model catalogue?